config/locales/en.yml

```diff
@@ -97,6 +97,9 @@ en:
       title: Allies
       outgoing: 'Outgoing Requests'
       incoming: 'Incoming Requests'
+      invited: 'Invited by email'
+      invited_on: 'Invited on: %{date}'
+      invitation_expired: 'Invitation expired?'
```

Comment:
For now you can go ahead and get translations for our other languages via Google Translate

Reply:
I think it would be simpler just to show `invitation_expired: 'Invitation expired'` if that's the case and nothing if it's not.